performancetest/src/main/java/org/corfudb/performancetest/LogUnitPerformanceTest.java

```diff
@@ -0,0 +1,119 @@
+package org.corfudb.performancetest;
+
+import com.codahale.metrics.MetricRegistry;
+import com.codahale.metrics.Timer;
+import io.netty.buffer.ByteBuf;
+import io.netty.buffer.Unpooled;
+import org.corfudb.protocols.wireprotocol.DataType;
+import org.corfudb.protocols.wireprotocol.LogData;
+import org.corfudb.runtime.CorfuRuntime;
+import org.corfudb.runtime.clients.LogUnitClient;
+import org.corfudb.util.serializer.Serializers;
+import org.junit.Test;
+import java.util.*;
+
+/**
+ * Created by Nan Zhang and Lin Dong.
+ */
+
+public class LogUnitPerformanceTest extends PerformanceTest{
+    static MetricRegistry metricRegistry;
+    Timer readTimer;
+    Timer writeTimer;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/app/annymartinez.m/CorfuDB/pullRequest?prid=4651454)